apps/dolly-frontend/src/main/js/src/components/fagsystem/tjenestepensjon/form/Form.js

```diff
@@ -18,38 +20,66 @@ export const initialYtelser = {
 	datoYtelseIverksattTom: null,
 }
 
-const initialOrdning = {
-	ordning: '',
+export const initialOrdning = {
+	ordning: '3010',
 	ytelser: [initialYtelser],
 }
 
-export const TjenestepensjonForm = ({ formikBag }) => (
-	<Vis attributt={path}>
-		<Panel
-			heading="Tjenestepensjon"
-			hasErrors={panelError(formikBag, path)}
-			iconType="pensjon"
-			startOpen={erForste(formikBag.values, [path])}
-			informasjonstekst={hjelpetekst}
-		>
-			<FormikDollyFieldArray name="pensjonforvalter.tp" header="Ordning" newEntry={initialOrdning}>
-				{(formPath, idx) => (
-					<React.Fragment key={idx}>
-						<React.Fragment>
-							<div className="flexbox--flex-wrap">
-								<FormikTextInput name={`${formPath}.ordning`} label="tp-nr" type="text" />
-								<YtelseForm
-									path={`${formPath}.ytelser`}
-									header="Ytelser"
-									initialYtelser={initialYtelser}
-								/>
-							</div>
+export const fetchTpOrdninger = () => {
+	if (!Options('tpOrdninger').length) {
+		const ordninger = PensjonApi.getTpOrdninger()
+		ordninger.then((json) => {
```

Comment:
Trenger vel ikke egen const for denne, kan bare gjøre PensjonApi.getTpOrdninger().then(...) siden getTpOrdninger kallet er såpass greit beskrevet 🤓

Reply:
fikset i "dolly-frontend/tp-bestillingssammendrag"